underscore.js

```diff
@@ -436,6 +436,12 @@
   _.toArray = function(obj) {
     if (!obj) return [];
     if (_.isArray(obj)) return slice.call(obj);
+    if (_.isString(obj)) {
+      // Keep surrogate pair characters together
+      // https://mathiasbynens.be/notes/javascript-unicode
+      var reStrSymbol = /[^\ud800-\udbff\udc00-\udfff]|[\ud800-\udbff][\udc00-\udfff]|[\ud800-\udbff\udc00-\udfff]/g;
```

Comment:
So the unicode character after `\udbff` is `\udc00`? My concern is that there are gaps between the two ranges and so collapsing it into one globbers the gaps.